scripts/project-updater.xsh

```diff
@@ -151,7 +151,7 @@ if not second_run:
     _git_status: CommandPipeline = {}
     _git_status = !(git -C @(project_folder) status)
 
-    if "fatal: not a git repository" in _git_status.err:
+    if _git_status.err != None and "fatal: not a git repository" in _git_status.err:
```

Comment:
in which case the `_git_status.err` is `None`?

Reply:
When `git -C @(project_folder) status` is a git repository